<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inserisci_cliente</class>
 <widget class="QWidget" name="inserisci_cliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuovo Cliente</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>381</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nome</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textNome"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Cognome</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textCognome"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Codice Fiscale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textCodiceFiscale"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Indirizzo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textIndirizzo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textEmail"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Telefono</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textTelefono"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Et√†</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textEta"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
